webapp/src/components/expanded_view/component.tsx

```diff
@@ -1025,19 +1044,17 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
                                 </Tooltip>
                             }
                         >
-                            <button
-                                className='button-close'
-                                style={this.style.closeViewButton}
+                            <CloseViewButton
+                                className='style--none'
```

Comment:
Do we need this className?

Reply:
I'd say yes, as with any other button, we don't want the default browser styling. Before we were overriding everything through `.button-close`. This is more consistent now.